client_documents_test.go

```diff
@@ -9,6 +9,12 @@ type docTest struct {
 	Name string `json:"name"`
 }
 
+type docTestBooks struct {
+	Book_id int    `json:"book_id"`
```

Comment:
Use  go convention, should be BookID.

Reply:
Indeed, thanks! Got mixed with the documents syntax :)